Description: Source /lib/lsb/init-functions, don't use it yet
Author: Peter Pentchev <roam@ringlet.net>
Forwarded: not-needed
Last-Update: 2014-06-13

--- a/tools/stunnel.init.in
+++ b/tools/stunnel.init.in
@@ -14,6 +14,8 @@
 #                    to completely disable *all* tunnels.
 ### END INIT INFO
 
+. /lib/lsb/init-functions
+
 DEFAULTPIDFILE="/var/run/stunnel4.pid"
 DAEMON=@bindir@/stunnel4
 NAME=stunnel
